core/src/test/java/com/exadel/etoolbox/linkinspector/core/services/data/impl/GridResourcesGeneratorImplTest.java

```diff
@@ -135,9 +137,10 @@ void setup() {
 
         context.registerInjectActivateService(
                 new MockHttpClientBuilderFactory(),
-                ImmutableMap.of(
-                        MockHttpClientBuilderFactory.PN_STATUS_CODE, HttpStatus.SC_NOT_FOUND,
-                        MockHttpClientBuilderFactory.PN_STATUS_MESSAGE, "Not Found")
+                Collections.unmodifiableMap(new HashMap<String, Object>() {{
```

Comment:
You don't need to wrap the `HashMap` into unmodifiable map. The service does not care if the arguments are modifiable or unmodifiable. Therefore you can save an extra operation